semantic.json

```diff
@@ -1,5 +1,5 @@
 {
-  "base": "public/semantic/",
+  "base": "public/semantic",
```

Comment:
So I made the /angular section to give myself a self-contained area, since I had no idea what was going on elsewhere.  I have only touched stuff there, in the img folder, and index.blade.php.  As far as I'm concerned, it's all fair game!